methods_to_test.js

```diff
@@ -0,0 +1,69 @@
+const capitalize = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+const reverseString = (string) => {
+    let splitString = string.split("")
```

Comment:
[OPTIONAL]
We should prefer the use of `const` over `let`. Had we used a linter this would have been given to us as a warning.